### ISSUE TITLE
It is time consuming to change our home page feature

### ISSUE THREAD

**Issue by Pinpickle** [hackersatcambridge/hac-website#99]
## What the enhancement is

Adding a way to change our feature on the home page without touching source code. Ideally we'd be able to give them an expiry date, so we can queue features up to change automatically.

## Why it could be helpful

This makes it easier to rapidly change features, which makes it less likely for the data we have to become stale.

In the future we could even generate it from our event stream, so it updates automatically.

A potential problem is the need to add images on the fly as well. We'll need some kind of s3 setup for this.

**Comment by jaredkhan** [comment]
Very related to #25 
I see two issues here:
- Building some controller that will handle what feature to show
- Building a nice UI for feeding information to that controller

The former will suffice for now. Could do this in about an hours work just having a list of Features somewhere and dates associated with them, controller just picks the first one it finds with a date in range.

**Comment by jaredkhan** [comment]
@Pinpickle Proposing we close this as with #25 closed it's no longer such a pain. Still missing the UI / database piece but we can leave that as an enhancement for future

**Comment by Pinpickle** [comment]
@jaredkhan I think it's still in the realm of time consuming as it's blocked by code-review. We could re-word this to say "We have to alter source code to change our feature" as a more general statement.

Then it can cover all the images we're throwing into our repo over time as that's not something we want to be doing long-term either (on account of repo size blowing up).

**Comment by jaredkhan** [comment]
Alright agreed. Lets reword and lower priority?

**Comment by moosichu** [comment]
I still think it's fairly high priority

As we have discovered today, Heroku might not rebuild our source very quickly, so having this be data-driven would be nice.

**Comment by jaredkhan** [comment]
I agree it would be nice but I don't think it's as important as getting the updates stream live

**Comment by Pinpickle** [comment]
Putting this into "priority medium" as I'd consider that our "fairly high priority" label

Main issues with build speed can be tackled by #89 

I don't think Heroku being unavailable is a very strong argument for action ([for context, the build system is down at the moment](https://status.heroku.com/incidents/1334)). If we followed that argument, we could say that Heroku going down completely (ie our website becoming unavailable) is a motivation for something like self-hosting. Alternatively, our DB provider could be having issues, giving us a motivation to move more stuff into source code.
